source/strings.rst

```diff
@@ -197,7 +202,7 @@ In Python 3, the concept of ``basestring`` makes no sense: text is only
 represented by ``str``.
 
 For type-checking text strings in code compatible with both versions, the
-:ref:`six` library offers ``string_types``, which is ``(basestring,)``
+:ref:`six <six>` library offers ``string_types``, which is ``(basestring,)``
```

Comment:
I'm not sure what this exactly does, so please be sure the result is as expected. (I can check the results, but not ATM.)

Reply:
In the original version, Sphinx uses the chapter name ("Compatibility library: six") as the link text.
In `six <six>`, the first thing is the link text and `<six>` specifies the target.

Actually, it might be better to use `six library <six>` everywhere: